ranger/data/scope.sh

```diff
@@ -290,6 +292,45 @@ handle_mime() {
             xls2csv -- "${FILE_PATH}" && exit 5
             exit 1;;
 
+        ## SQLite
+        *sqlite3)
+            ## Preview as text conversion
+            local tables=""
```

Comment:
POSIX sh doesn't support local variables. Feel free to make up a prefix that's clearly unique to the sqlite previews.

Reply:
Refactored.